crowbar_framework/lib/schema_migration.rb

```diff
@@ -143,7 +152,11 @@ def self.migrate_object(bc_name, template, all_scripts, attributes, deployment)
     scripts.each do |script|
       # we only pass attributes and deployment to not encourage direct access
       # to the proposal
-      attributes, deployment = run_script(script, is_upgrade, template['attributes'][bc_name], template['deployment'][bc_name], attributes, deployment)
+      begin
+        attributes, deployment = run_script(script, is_upgrade, template['attributes'][bc_name], template['deployment'][bc_name], attributes, deployment)
+      rescue StandardError => e
+        raise "error while executing migration script #{script}:\n#{e.message}"
```

Comment:
That's really because the other messages are actually displayed in this string (`e.message`), iirc. I don't remember the details for sure; the best way to figure it out is to test the patch :-)